packages/matchbox/src/components/Radio/styles.js

```diff
@@ -0,0 +1,69 @@
+import { tokens } from '@sparkpost/design-tokens';
+import { Box } from '../Box';
+import { visuallyHidden } from '../../styles/helpers';
+import styled from 'styled-components';
+import { margin } from 'styled-system';
+
+export const Wrapper = styled('div')`
+  ${margin}
+  position: relative;
+  border: none;
+  padding: 0;
+`;
+
+export const StyledLabel = styled('label')`
+  display: flex;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  white-space: nowrap;
+  user-select: none;
+
+  &:hover {
+    cursor: ${props => (props.disabled ? 'not-allowed' : 'pointer')};
+  }
+`;
+
+const commonCheckedStyles = `
+  position: absolute;
+  top: 0;
+  left: 0;
+  transition: ${tokens.motionDuration_fast} ${tokens.motionEase_in_out};
```

Comment:
It's better not to implicitly target `all` - better to specify which properties will be transitioning - looks like `opacity` and/or maybe `fill`?